README.md

```diff
@@ -210,6 +211,28 @@ For dex.decred.org, reach out
 [on Element](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org?via=decred.org&via=matrix.org&via=planetdecred.org)
 to appeal.
 
+## Fees
+
+The dex does not charge trading fees, but users pay on-chain transaction fees.
+Transaction fees vary based on how orders are matched.
+
+To ensure that on-chain transaction fees do not eat a significant portion of the
+order quantity, orders must be specified in increments of a minimum lot size.
+To illustrate, if on-chain transaction fees worked out to $5, and a user was able
+to place an order to trade $10, they would lose half of their trade to
+transaction fees. For chains with single-match fees of $5, if the operator wanted
+to limit possible fees to under 1% of the trade, the minimum lot size would need
+to be set to about $500.
+
+The scenario with the lowest fees is for an entire order to be consumed by a
+single match. If this happens, the user pays 2 transaction fees: one swap and
+one redemption. The worst case is for the order to be consumed by one single-lot
```

Comment:
> If this happens, the user pays 2 transaction fees: one swap and one redemption.

In the above, I would state that these are on different chains.  That's important because if one asset is low fee like DCR, and the other is high like BTC, you really only care about the tx on the high fee chain.